sdk/storage/Azure.ResourceManager.Storage/api/Azure.ResourceManager.Storage.netstandard2.0.cs

```diff
@@ -229,8 +229,8 @@ public partial class FileService : Azure.ResourceManager.Core.ArmResource
     public partial class FileServiceData : Azure.ResourceManager.Models.ResourceData
     {
         public FileServiceData() { }
-        public Azure.ResourceManager.Storage.Models.CorsRules Cors { get { throw null; } set { } }
-        public Azure.ResourceManager.Storage.Models.ProtocolSettings ProtocolSettings { get { throw null; } set { } }
+        public System.Collections.Generic.IList<Azure.ResourceManager.Storage.Models.CorsRule> CorsRulesValue { get { throw null; } }
+        public Azure.ResourceManager.Storage.Models.SmbSetting ProtocolSmb { get { throw null; } set { } }
```

Comment:
we drop certain keywords since they make the name longer and awkward, `Settings` is one of those.  The full list is here
https://github.com/Azure/autorest.csharp/pull/1992/files#diff-503f36a55deabaa9bb8002d4a53adfb91520631e75dc409b27265eb45d0e2f53R225

If the name is awkward here we should handle with property name directive